packages/cozy-sharing/src/SharingProvider.jsx

```diff
@@ -150,7 +152,16 @@ export class SharingProvider extends Component {
       updateSharingInStore(this.dispatch, newSharing)
     } else {
       const docsId = getSharingDocIds(newSharing)
-      createSharingInStore(client, doctype, this.dispatch, docsId, newSharing)
+      this.state.synchronousJobQueue.push({
```

Comment:
Obligé de mettre ça dans le state ? On peut pas avoir une variable de classe plutôt ? `this.synchronousJobQueue` ?